cmd/lk/agent.go

```diff
@@ -330,6 +368,91 @@ func createAgentClient(ctx context.Context, cmd *cli.Command) (context.Context,
 	return ctx, nil
 }
 
+func initAgent(ctx context.Context, cmd *cli.Command) error {
+	// TODO: (@rektdeckard) move compatibility flag into template index,
+	// then show template picker containing only compatible templates
+	if !(cmd.IsSet("lang") || cmd.IsSet("template") || cmd.IsSet("template-url")) {
+		var lang string
+		// Prompt for language
+		if err := huh.NewSelect[string]().
+			Title("Select the language for your agent project").
+			Options(
+				huh.NewOption("Python", "python"),
+				huh.NewOption("Node.js", "node"),
+			).
+			Value(&lang).
+			WithTheme(util.Theme).
+			Run(); err != nil {
+			return err
+		}
+
+		switch lang {
+		case "node":
+			templateURL = "https://github.com/livekit-examples/agent-starter-node"
+		case "python":
+			templateURL = "https://github.com/livekit-examples/agent-starter-python"
+		default:
+			return fmt.Errorf("unsupported language: %s", lang)
+		}
+	}
+
+	logger.Debugw("Initializing agent project", "working-dir", workingDir)
+
+	// Create sandbox
+	if !cmd.Bool("no-sandbox") || sandboxID == "" {
+		if err := util.Await("Creating sandbox app...", ctx, func(ctx context.Context) error {
+			token, err := requireToken(ctx, cmd)
+			if err != nil {
+				return err
+			}
+
+			appName = cmd.Args().First()
+			if appName == "" {
+				appName = project.Name
+			}
+			// We set agent name in env for use in template tasks
+			os.Setenv("LIVEKIT_AGENT_NAME", appName)
+
+			// TODO: (@rektdeckard) figure out why AccessKeyProvider does not immediately
+			// have access to newly-created API keys, then remove this sleep
+			time.Sleep(4 * time.Second)
```

Comment:
maybe a future todo, this is a long sleep though. maybe a loop to check for the keys?